db/config.ts

```diff
@@ -1,6 +1,20 @@
 import { defineDb } from 'astro:db';
 
 // https://astro.build/db/config
+
+const ExampleTable = defineTable({
+    columns: {
```

Comment:
I think this is left over from the demo we had a few weeks ago. Since these are just example table it should be remove.

Reply:
You can actually rework this example table into the email log table